runtime/jvmti/jvmtiHook.c

```diff
@@ -838,20 +841,22 @@ jvmtiHookClassLoad(J9HookInterface** hook, UDATA eventNum, void* eventData, void
 {
 	J9VMClassLoadEvent * data = eventData;
 	J9JVMTIEnv * j9env = userData;
+	J9VMThread * currentThread = data->currentThread;
 
 	Trc_JVMTI_jvmtiHookClassLoad_Entry();
 
 	ENSURE_EVENT_PHASE_START_OR_LIVE(jvmtiHookClassLoad, j9env);
 
 	/* Do not report the event for arrays or primitive types */
 
-	if (J9ROMCLASS_IS_PRIMITIVE_OR_ARRAY(data->clazz->romClass) == 0) {
+	if (J9ROMCLASS_IS_PRIMITIVE_OR_ARRAY(data->clazz->romClass) == 0
```

Comment:
```suggestion
	if ((0 == J9ROMCLASS_IS_PRIMITIVE_OR_ARRAY(data->clazz->romClass))
```

Reply:
Please stop marking comments as "resolved" without either explaining why the feedback is inappropriate or making the suggested change.